docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,96 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, and show off your metrics to your team or visitors.
```

Comment:
@marcusolsson - What permissions are required to add, edit, and delete playlists? What about start, stop, and control?

I ask because I cannot add, edit, or delete them in the playground, so I am guessing you need something like Admin rights for the former and Guest or User rights for the latter.

Reply:
I believe you need to have the [Editor role](http://localhost:3004/permissions/organization_roles/#editor-role) to create and modify playlists.

Anyone with access to Grafana should be able to start and control playlists.

Reply:
I also don't like the Dashboard of other Dashboards. I'm not sure how else to describe it though; a playlist is simply a presentation of dashboards in a sequence. Like a PowerPoint of dashboards, which are full of data / metrics.

I used your suggestion, with this change "... or to **present** your metrics to your team or visitors." A dashboard in itself already _displays_ your metrics. A Playlist plays them, or presents them.